tests/test_bundle_ckpt_export.py

```diff
@@ -37,67 +38,69 @@ def setUp(self):
             os.environ["CUDA_VISIBLE_DEVICES"] = "0"  # default
+        self.meta_file = os.path.join(os.path.dirname(__file__), "testing_data", "metadata.json")
+        self.config_file = os.path.join(os.path.dirname(__file__), "testing_data", "inference.json")
+        self.tempdir_obj = tempfile.TemporaryDirectory()
+        tempdir = self.tempdir_obj.name
+        self.def_args = {"meta_file": "will be replaced by `meta_file` arg"}
+        self.def_args_file = os.path.join(tempdir, "def_args.yaml")
+
+        self.ckpt_file = os.path.join(tempdir, "model.pt")
+        self.ts_file = os.path.join(tempdir, "model.ts")
+
+        self.parser = ConfigParser()
+        self.parser.export_config_file(config=self.def_args, filepath=self.def_args_file)
+        self.parser.read_config(self.config_file)
+        self.net = self.parser.get_parsed_content("network_def")
 
     def tearDown(self):
         if self.device is not None:
             os.environ["CUDA_VISIBLE_DEVICES"] = self.device
         else:
             del os.environ["CUDA_VISIBLE_DEVICES"]  # previously unset
+        self.tempdir_obj.cleanup()
 
     @parameterized.expand([TEST_CASE_1, TEST_CASE_2, TEST_CASE_3])
-    def test_export(self, key_in_ckpt, use_trace):
-        meta_file = os.path.join(os.path.dirname(__file__), "testing_data", "metadata.json")
-        config_file = os.path.join(os.path.dirname(__file__), "testing_data", "inference.json")
-        with tempfile.TemporaryDirectory() as tempdir:
-            def_args = {"meta_file": "will be replaced by `meta_file` arg"}
-            def_args_file = os.path.join(tempdir, "def_args.yaml")
-
-            ckpt_file = os.path.join(tempdir, "model.pt")
-            ts_file = os.path.join(tempdir, "model.ts")
-
-            parser = ConfigParser()
-            parser.export_config_file(config=def_args, filepath=def_args_file)
-            parser.read_config(config_file)
-            net = parser.get_parsed_content("network_def")
-            save_state(src=net if key_in_ckpt == "" else {key_in_ckpt: net}, path=ckpt_file)
-
-            cmd = ["coverage", "run", "-m", "monai.bundle", "ckpt_export", "network_def", "--filepath", ts_file]
-            cmd += ["--meta_file", meta_file, "--config_file", f"['{config_file}','{def_args_file}']", "--ckpt_file"]
-            cmd += [ckpt_file, "--key_in_ckpt", key_in_ckpt, "--args_file", def_args_file]
-            if use_trace == "True":
-                cmd += ["--use_trace", use_trace, "--input_shape", "[1, 1, 96, 96, 96]"]
-            command_line_tests(cmd)
-            self.assertTrue(os.path.exists(ts_file))
-
-            _, metadata, extra_files = load_net_with_metadata(
-                ts_file, more_extra_files=["inference.json", "def_args.json"]
-            )
-            self.assertIn("schema", metadata)
-            self.assertIn("meta_file", json.loads(extra_files["def_args.json"]))
-            self.assertIn("network_def", json.loads(extra_files["inference.json"]))
```

Comment:
This part of the tests, loading the TorchScript object back and inspecting properties, now isn't being done. I think this was useful to making sure the produced results were correct.